lib/screens/login.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:admu_student_app/screens/onboarding.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class Login extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MediaQuery(
+      data: MediaQueryData(),
+      child: MaterialApp(
+        theme: ThemeData(
+            textTheme:
+                GoogleFonts.dmSansTextTheme(Theme.of(context).textTheme)),
+        home: Directionality(
+          textDirection: TextDirection.ltr,
+          child: Scaffold(
+            body: Center(
+              child: Container(
+                //Parent Column
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    CircleAvatar(
+                      backgroundImage: AssetImage('onboarding.png'),
+                      radius: 100,
+                    ),
+                    Text('Welcome to Beadle!',
```

Comment:
text isn't centered (visible in smaller screens)
![image](https://user-images.githubusercontent.com/72055285/116119493-14ba1280-a6f1-11eb-9a22-faf0260538fd.png)